jest.setup.redis-mock.js

```diff
@@ -1,8 +1,19 @@
 jest.mock('ioredis', () => {
-    const mPool = {
-        get: jest.fn(),
-        set: jest.fn(),
-        disconnect: jest.fn(),
+    const Redis = require('ioredis-mock')
+    if (typeof Redis === 'object') {
```

Comment:
Why does this check type? Won't it always be the same?

Reply:
I added a link describing where I got the code from... And fixed the other link.